sdk/include/psdk/winbase.h

```diff
@@ -189,13 +191,28 @@ extern "C" {
 #define CREATE_FORCEDOS			0x00002000
 #define BELOW_NORMAL_PRIORITY_CLASS	0x00004000
 #define ABOVE_NORMAL_PRIORITY_CLASS	0x00008000
+#if (_WIN32_WINNT >= _WIN32_WINNT_VISTA)
+#define INHERIT_PARENT_AFFINITY             0x00010000
+// TODO: When waa it deprecated?
```

Comment:
??

Reply:
As in, It's marked as deprecated in SDK 10, but I don't know when that happened (earlier)...